### PR TITLE
update build instructions

### PR DESCRIPTION
**Pull request by dsvoronin**
#134 suggestion, also sent a PR to gojuno/commander#15 for meaningful message about missing ANDROID_HOME

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
 
 ### How to build
 
-Dependencies: you only need `docker` and `bash` installed on your machine.
+#### To build composer
+
+environmental variable ANDROID_HOME must be set.
```
Comment (artem-zinnatullin) [inline README.md]: Capital `E`, wrap `ANDROID_HOME` with ` :)

File: README.md [hunk 2]
```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
...
+
+you need `npm` and `nodejs` packages installed.
+
+```console
+cd html-report
+npm install
+npm build
+```
+
+#### All-in-one script (used in Travis build)
```
Comment (artem-zinnatullin) [inline README.md]: Can we put this one at the top of "How to build"? I don't want people install npm and stuff to build Composer

File: README.md [hunk 3]
```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
...
-Dependencies: you only need `docker` and `bash` installed on your machine.
+#### To build composer
+
+environmental variable ANDROID_HOME must be set.
+
+```console
+./gradlew build
+```
+
+#### To build html-report
```
Comment (artem-zinnatullin) [inline README.md]: "Build HTML report module"?

File: README.md [hunk 4]
```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
 
 ### How to build
 
-Dependencies: you only need `docker` and `bash` installed on your machine.
+#### To build composer
```
Comment (artem-zinnatullin) [inline README.md]: Build Composer

File: README.md (hunk 1 above)
Comment (artem-zinnatullin) [reply README.md]: And afaik it's called environment variable, not environmental hehe

File: README.md [hunk 5]
```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
...
+
+```console
+cd html-report
+npm install
+npm build
+```
+
+#### All-in-one script (used in Travis build)
+
+you only need `docker` and `bash` installed on your machine.
```
Comment (artem-zinnatullin) [inline README.md]: Dependencies: `docker` and `bash`.

File: README.md [hunk 6]
```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
...
+
+environmental variable ANDROID_HOME must be set.
+
+```console
+./gradlew build
+```
+
+#### To build html-report
+
+you need `npm` and `nodejs` packages installed.
```
Comment (artem-zinnatullin) [inline README.md]: Dependencies: `npm` and `nodejs`.

File: README.md (hunk 1 above)
Comment (dsvoronin) [reply README.md]: :greenpeace:

File: dependencies.gradle [hunk 7]
```diff
@@ -3,7 +3,7 @@ ext.versions = [
 
         rxJava           : '1.3.0',
         jCommander       : '1.71',
-        commander        : '0.1.6',
+        commander        : '0.1.7',
```
Comment (artem-zinnatullin) [inline dependencies.gradle]: why in this pr tho? pls extract to separate one

Comment (dsvoronin) [reply dependencies.gradle]: because it addresses the same thing: informative message on ANDROID_HOME error while running build, and no more

**Pull request merged**